src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,72 @@
+package duke;
+
+import java.io.*;
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file
+ *
+ * @author sibinhho99-nus
+ */
+public class Storage {
+    public TaskList taskList = new TaskList();
+
+    /**
+     * Loads the task list from storage
+     */
+    public void load() throws IOException {
+        FileReader fr = new FileReader("storage.txt");
+        BufferedReader bufferedReader = new BufferedReader(fr);
+        while (bufferedReader.ready()) {
+            String line = bufferedReader.readLine();
+
+            if (line.charAt(3) == 'D') {
```

Comment:
what happens when you save a `TaskList` with more than 9 tasks, then try to load it? will this still work as expected?